recipes/bdwgc/all/conanfile.py

```diff
@@ -151,7 +151,7 @@ def package_info(self):
         if self.options.gc_debug:
             self.cpp_info.components["gc"].defines.append("GC_DEBUG")
         if self.settings.os == "Windows":
-            self.cpp_info.components["gc"].requires = ["libatomic_ops::libatomic_ops"]
+            self.cpp_info.components["gc"].requires = ["Atomic_ops::atomic_ops"]
```

Comment:
```suggestion
            self.cpp_info.components["gc"].requires = ["libatomic_ops::atomic_ops"]
```
It's the package_name::component_name: https://docs.conan.io/en/latest/creating_packages/package_information.html#using-components

Reply:
Done